test/ngAnimate/animateCssSpec.js

```diff
@@ -2092,6 +2092,29 @@ describe("ngAnimate $animateCss", function() {
           expect(element.css('transition-property')).toMatch('color');
           expect(style).toContain('ease-in');
         }));
+
+        it("should only execute the animation if any CSS to styles are mixed into together",
```

Comment:
should execute the animation only if there is any CSS styling to go with the transition